auth/src/main/java/org/aerogear/mobile/security/AsyncSecurityCheckExecutor.java

```diff
@@ -33,7 +33,7 @@
     private final static Logger LOG = MobileCore.getLogger();
 
     /**
-     * Custom {@link ExecutorService} for AsyncSecurityCheckExecutor.
+     * Custom {@link ExecutorService} for AsyncSecurityCheckExecutor. Can be null.
```

Comment:
It can't be null. But, as said in another comment, this kind of information should be put in methods/constructors that assign a value to this member variable. Here you should only describe what the member variable is used for.